src/components/AndroidStatusBar.tsx

```
@@ -0,0 +1,102 @@
+          ClawBox
+        </span>
+      </div>
+
+      {/* Right side - Status icons and time */}
+      <div className="flex items-center gap-3">
+        {/* WiFi icon */}
+        <svg
+          className="w-4 h-4 text-white/90"
+          viewBox="0 0 24 24"
+          fill="currentColor"
+        >
```

Biome (2.4.6)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)